secret_number.js

```diff
@@ -1,4 +1,7 @@
 'use strict';
 module.exports = function() {
-  
-};
\ No newline at end of file
+  let secretNumber = ((Math.random()*1000001));
```

Comment:
The secretNumber should be an integer. This solution would allow a decimal number to be created.
Fix it so the secretNumber is always an integer.